scripts/run_collector.py

```diff
@@ -1,14 +1,19 @@
 import os
 
 from api.ocs import read_ocs_zipfile, OcsProgramProvider
+from common.output import print_collector_info
```

Comment:
I assume a `print_selector_info` is going to be added or we can just put that inside the collector